c10/core/DispatchKeySet.h

```diff
@@ -108,6 +116,11 @@ class DispatchKeySet final {
     // didn't do it for now.
     return static_cast<DispatchKey>(64 - llvm::countLeadingZeros(repr_));
   }
+
+  DispatchKey highestPriorityBackendTypeId() const {
+    return (*this & DispatchKeySet(DispatchKeySet::FULL_AFTER_INCLUSIVE, DispatchKey::PrivateUse3))
```

Comment:
Might be worth having an alias defined alongside the dispatch keys instead of using `PrivateUse3` directly, just to encapsulate.